include/exadg/incompressible_navier_stokes/spatial_discretization/operator_coupled.cpp

```diff
@@ -1219,7 +1221,7 @@ OperatorCoupled<dim, Number>::apply_inverse_negative_laplace_operator(VectorType
 
       if(laplace_operator->operator_is_singular())
       {
-        set_zero_mean_value(vector_zero_mean);
```

Comment:
But doesn't this mean that you can now remove the function `set_zero_mean_value()`? This does not seem to be implemented in deal.II.